rcaracaus/pledge-vegan-node
# 15-vegan-detail (3)[2]

Steps to test:

1. Go to link localhost:3000/vegans/kathy-freston
2. Check that a page exact the same as pledgevegan.com/vegans/kathy-freston is seen.
3. Pull 14-famous-vegans into your local repo.
4. Go to famous vegans page localhost:3000/famous-vegans
5. Click on one of the vegans and check the page is the same as the pledgevegan.com site.
